From dabb39cb7f7c767f39d0f99493956f0ad8a247ed Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 26 Nov 2024 17:38:31 +0100
Subject: [PATCH] Make launch script executable

Instead of setting the install mode to executable only for root, make the
launch script executable for all users. Without this, users cannot launch
the application if installed on the system.
---
 errands/errands.py  | 0
 errands/meson.build | 1 -
 meson.build         | 1 -
 3 files changed, 2 deletions(-)
 mode change 100644 => 100755 errands/errands.py

diff --git a/errands/errands.py b/errands/errands.py
old mode 100644
new mode 100755
diff --git a/errands/meson.build b/errands/meson.build
index 2ac42477..3e46e12f 100644
--- a/errands/meson.build
+++ b/errands/meson.build
@@ -4,7 +4,6 @@ configure_file(
   configuration: conf,
   install: true,
   install_dir: get_option('bindir'),
-  install_mode: 'r-xr--r--'
 )
 
 install_data('application.py', install_dir: moduledir)
diff --git a/meson.build b/meson.build
index 37b11eaa..48afd625 100644
--- a/meson.build
+++ b/meson.build
@@ -53,7 +53,6 @@ configure_file(
   configuration: conf,
   install: true,
   install_dir: get_option('bindir'),
-  install_mode: 'r-xr--r--',
 )
 install_data(join_paths('errands', 'application.py'), install_dir: moduledir)
 install_data(join_paths('errands', 'state.py'), install_dir: moduledir)
